packages/server/src/api/controllers/pools.ts

```diff
@@ -45,17 +39,19 @@ const getTopPoolsValidator = celebrate({
   })
 });
 
-async function getTopPools(req: Request<unknown, unknown, unknown, GetTopPoolsQuery>) {
-  const { count } = req.query;
-  return cached.getTopPools(count);
+async function getTopPools(req: Request) {
+  // TODO: add override for route.get()
+  // Request<any, any, any, ParsedQs> query must be a ParsedQs
+  // We should add a union type for all validated queries
+  // or find a better TS integration with Celebrate
+  const { count }: GetTopPoolsQuery = <any> req.query;
```

Comment:
Want to clean this type casting up by adding an additional override to `route.get` and other http method fns. Could also tackle this by wrapping route.get and casting in there.